src/Entities/Entity.h

```diff
@@ -562,6 +580,9 @@ class cEntity
 	/** Called when the specified player right-clicks this entity */
 	virtual void OnRightClicked(cPlayer & a_Player) {}
 
+	/** Called when the specified player interact at specific coords of this entity. Depend of entity and mostly used by ArmorStand */
+	virtual void OnClickedAt(cPlayer & a_Player, Vector3f a_TargetPos, eHand a_Hand) {}
```

Comment:
Is this right-click, left-click or any-click?

Reply:
Undefined / Both
https://wiki.vg/index.php?title=Protocol&oldid=14204#Use_Entity
https://github.com/Buom01/cuberite/blob/armor-stand/src/Protocol/Protocol_1_9.cpp#L2853
As you see in the code, that's happen when the player use an entity with coords. Nothing more.
The doc says both, but in practice I don't remember.